src/components/common/NameRequestInfo.vue

```diff
@@ -19,26 +22,44 @@
             <span class="val">{{ expirationDate }}</span>
           </li>
           <li id="status">
-            <v-icon v-if="state === NameRequestStates.APPROVED ||
-                          state === NameRequestStates.CONDITIONAL"
-              color="green" class="nr-status-icon">mdi-check</v-icon>
+            <v-icon
+              v-if="state === NameRequestStates.APPROVED ||
+                state === NameRequestStates.CONDITIONAL"
```

Comment:
This is a good example why I prefer parentheses around more complex if statements for ease of reading/reviewing/understanding.

Reply:
This is actually not a bad one. You can leave it as-is.